ihmc-state-estimation/src/test/java/us/ihmc/stateEstimation/humanoid/kinematicsBasedStateEstimation/PelvisRotationalStateUpdaterTest.java

```diff
@@ -121,7 +123,9 @@ public void testInitializeAndReadWithOneIMU()
       PelvisRotationalStateUpdaterInterface pelvisRotationalStateUpdater = new IMUBasedPelvisRotationalStateUpdater(inverseDynamicsStructure, imuSensors, 1.0e-3, registry);
       new DefaultParameterReader().readParametersInRegistry(registry);
 
-      
+      // This must be false, otherwise the test fails.
+      ((YoBoolean) registry.findVariable("zeroEstimatedRootYawAtInitialization")).set(false);
```

Comment:
When this variable is true, the test fails, because the everything in the future is done relative to that initial yaw.

Reply:
I think the comment should contain this information.